docs/src/main/docs/microprofile/02_server-configuration.adoc

```diff
@@ -57,23 +58,29 @@ ssl:
     keystore-passphrase: "abcd"
 ----
 
+== Configure "admin" port
+For this example, we will use a `yaml` file. We configure a port 8011 that would server health and metrics endpoints,
```

Comment:
`would server health`, this looks like a typo.

Reply:
I think here, where the doc _first_ introduces additional ports, it should mention that such additional ports are restricted in what they can do. The restrictions can be explained in more detail later (see my note below).

Reply:
I have split this into two parts

- overall explanation of ports in MP and restrictions
- example description